### ISSUE TITLE
Install Error

### ISSUE THREAD

**Issue by Batmanabcdefg** [JuliaFEM/UMAT.jl#19]
**I cannot install UMAT.Jl in Julia 1.2.0**

```
ERROR: Unsatisfiable requirements detected for package BinaryProvider [b99e7846]:
 BinaryProvider [b99e7846] log:
 ├─possible versions are: 0.5.3 or uninstalled
 └─restricted to versions 0.5.6-0.5 by UMAT [d4351ba0] — no versions left
   └─UMAT [d4351ba0] log:
     ├─possible versions are: 0.1.0 or uninstalled
     └─UMAT [d4351ba0] is fixed to version 0.1.0
```

**Comment by TeroFrondelius** [comment]
I would try `] update`

**Comment by Batmanabcdefg** [comment]
I just use this method, but it does not solve the problem```

ERROR: Unsatisfiable requirements detected for package BinaryProvider [b99e7846]:
 BinaryProvider [b99e7846] log:
 ├─possible versions are: 0.5.3 or uninstalled
 └─restricted to versions 0.5.6-0.5 by UMAT [d4351ba0] — no versions left
   └─UMAT [d4351ba0] log:
     ├─possible versions are: 0.1.0 or uninstalled
     └─UMAT [d4351ba0] is fixed to version 0.1.0
```

**Comment by TeroFrondelius** [comment]
Next thing to try: activate an empty environment and add just the UMAT. If you figure out the problem,  pull request is much appreciated.

**Comment by Batmanabcdefg** [comment]
@TeroFrondelius  I have just reinstalled JuliaPro 1.2.0 ... add  BinaryProvider & umat.jl

however, it doesnot work.,...... the same error ...

**Comment by Batmanabcdefg** [comment]
@ahojukka5  this UMAT problem should depend on  BinaryProvider, i.e.

```
ERROR: Unsatisfiable requirements detected for **package BinaryProvider** [b99e7846]:
BinaryProvider [b99e7846] log:
├─possible versions are: 0.5.3 or uninstalled
└─restricted to versions 0.5.6-0.5 by UMAT [d4351ba0] — no versions left
└─UMAT [d4351ba0] log:
├─possible versions are: 0.1.0 or uninstalled
└─UMAT [d4351ba0] is fixed to version 0.1.0
```

**Comment by ahojukka5** [comment]
Yes, this very likely has nothing to do with `FEMBasis.jl` dependency issue.

**Comment by TeroFrondelius** [comment]
> JuliaPro 1.2.0 

Don't use JuliaPro it is not meant for open source development. It is meant for corporate users behind firewalls and it only works with curated packages, which doesn't include any JuliaFEM packages.
